### PR TITLE
chore(deps): Fix multi arch images errors

### PR DESCRIPTION
**Pull request by Bonnai2112**

### CODE REVIEW

File: bitcoind/Dockerfile [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
+	&& wget -qO bitcoin.tar.gz ${BITCOIN_URL} \
+	&& echo "${BITCOIN_SHA256} bitcoin.tar.gz" | sha256sum -c - \
+	&& mkdir bin \
+	&& tar -xzvf bitcoin.tar.gz -C /tmp/bin --strip-components=2 "bitcoin-${BITCOIN_VERSION}/bin/bitcoin-cli" "bitcoin-${BITCOIN_VERSION}/bin/bitcoind"
+
+FROM debian:buster-slim
+COPY --from=builder "/tmp/bin" /usr/local/bin
+
+EXPOSE 8332 8333 18332 18333 18443 18444
+CMD ["bitcoind"]
```
Comment (Bladrak) [inline bitcoind/Dockerfile:26]: Missing blank line at the end

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
 Versions
 ========
 
+2022-04-25
+----------
+* bumping image version for bitcoind to 23.0
+
+2022-04-21
```
Comment (Bladrak) [inline CHANGELOG.md:8]: You can remove this now :)

**Review by Bladrak — CHANGES_REQUESTED** [review]

File: bitcoind/Dockerfile (hunk 1 above)
Comment (Bonnai2112) [reply bitcoind/Dockerfile]: done

File: CHANGELOG.md (hunk 2 above)
Comment (Bonnai2112) [reply CHANGELOG.md]: done

File: bitcoind/Dockerfile [hunk 3]
```diff
@@ -2,21 +2,17 @@
 FROM debian:buster-slim as builder
 
 ARG BITCOIN_VERSION
-ARG BITCOIN_SHA256
-
-ENV BITCOIN_URL https://bitcoin.org/bin/bitcoin-core-${BITCOIN_VERSION}/bitcoin-${BITCOIN_VERSION}-x86_64-linux-gnu.tar.gz
+ARG SHA256_AMD64
```
Comment (Bladrak) [inline bitcoind/Dockerfile:5]: You could have only one arg BITCOIN_SHA256 and switch it in the build config, based on the arch?

File: bitcoind/config.yml [hunk 4]
```diff
@@ -1,8 +1,10 @@
...
     build_args:
-      BITCOIN_VERSION: 0.21.1
-      BITCOIN_SHA256: 366eb44a7a0aa5bd342deea215ec19a184a11f2ca22220304ebb20b9c8917e2b
+      BITCOIN_VERSION: 23.0
+      SHA256_ARM64: 06f4c78271a77752ba5990d60d81b1751507f77efda1e5981b4e92fd4d9969fb
+      SHA256_AMD64: 2cca490c1f2842884a3c5b0606f179f9f937177da4eadd628e3f7fd7e25d26d0
     test_config:
       cmd:
         - bitcoind --version
+    platforms: *base_platforms
```
Comment (Bladrak) [inline bitcoind/config.yml:10]: Here you can have a 23.0_amd and a 23.0_arm config with different SHAs

**Review by Bladrak — CHANGES_REQUESTED** [review]

File: bitcoind/config.yml (hunk 4 above)
Comment (Bonnai2112) [reply bitcoind/config.yml:10]: Not need, it is when you use this image in docker run or docker service , Docker picks the correct image based on the node's platform by setting the --platform flag to specify the target platform for the build output, (for example, linux/amd64, linux/arm64, or darwin/amd64). For more information you can see this link https://docs.docker.com/buildx/working-with-buildx/

File: bitcoind/Dockerfile (hunk 3 above)
Comment (Bonnai2112) [reply bitcoind/Dockerfile:5]: Done !

File: dind/config.yml [hunk 5]
```diff
@@ -1,19 +1,17 @@
...
       TASKFILE_VERSION: *TASKFILE_VERSION
     test_config:
       cmd:
-        - *ci_helper_test
         - aws --version
         - az version
         - docker --version
         - docker-compose --version
-        - task --version
+        #- task --version
```
Comment (Bladrak) [inline dind/config.yml:15]: ?

File: bitcoind/Dockerfile (hunk 3 above)
Comment (Bladrak) [reply bitcoind/Dockerfile:5]: How is it done?

File: bitcoind/config.yml (hunk 4 above)
Comment (Bladrak) [reply bitcoind/config.yml:10]: This was in reference of this comment: https://github.com/ekino/docker-buildbox/pull/764/files#r867968980

**Review by Bladrak — CHANGES_REQUESTED** [review]

File: dind/config.yml (hunk 5 above)
Comment (Bonnai2112) [reply dind/config.yml:15]: Oups... it is a mistake ^^

**Review by Bladrak — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,16 @@
 Versions
 ========
+
+2022-23-06
+----------
+* Pass specific build_args for each arch
```
Comment (Bladrak) [inline CHANGELOG.md:6]: This should be a bit more of a functional than technical description. Something around the lines of `Handle multi-arch build for all images` would probably be best. You can also detail the version bumps

**Review by Bladrak — CHANGES_REQUESTED** [review]

**Review by Bladrak — APPROVED** [review]

**Review by Bladrak — CHANGES_REQUESTED** [review]

**Pull request closed without merging**